tests/test_app/TestApp/Locale/rule_1_po/default.po

```diff
@@ -18,7 +18,8 @@ msgstr "Plural Rule 1 (translated)"
 msgctxt ""
 "This is the context"
 msgid "%d = 1"
-msgstr "First Context trasnlation"
+msgstr ""
+"First Context trasnlation"
```

Comment:
Typo in 'translation'

Reply:
My bad, I just copied the typo :-). Both are fixed in latest commit.